bodhi/server/services/releases.py

```diff
@@ -332,3 +337,75 @@ def save_release(request):
     request.db.flush()
 
     return r
+
+
+@critpath_packages.get(permission='edit', renderer='json',
+                       error_handler=bodhi.server.services.errors.json_handler,
+                       validators=(colander_body_validator, validate_release_name)
+                       )
+def get_critpath_packages(request):
+    """
+    Get a list of critical path packages for a specific release.
+
+    Args:
+        request (pyramid.request): The current request.
+    Returns:
+        dict: The dictionary {'packages': critpath_packages}.
+    """
+    data = request.validated
+
+    release = data.pop('release', None)
```

Comment:
You don't need to ```pop``` this - you can just use ```request.validated['release']``` directly below and save two lines of code.